scripts/buildccf.sh

```diff
@@ -66,4 +66,6 @@ $buildccf_sh
 
 # Remove all files associated with build
 rm -rf $tempdir
-exec bash
\ No newline at end of file
+exec bash
+
+echo "Getting Python dependencies"
```

Comment:
Why is this file getting changed?

Reply:
I added a pic install for requirements.txt. Wasn't sure where to put it so I put it here for now

Reply:
This is the build script for the core of CCF-VM, nothing in here should change for this PR